specification/marketplaceordering/resource-manager/Microsoft.MarketplaceOrdering/stable/2015-06-01/Agreements.json

```diff
@@ -500,20 +551,30 @@
       },
       "description": "Old Agreement Terms definition"
     },
+    "UnsupportedMediaTypeErrorResponse": {
+      "description": "Error response that indicates the media type in the request is unsupported.",
+      "type": "object",
+      "properties": {
+        "Message": {
```

Comment:
Fix the LintDiff.

Reply:
LintDiff says this property needs to be called "message" according to the camelcase guidance, however, I cant change this, since the swagger must match the code, and the code sends "Message" and cant be changed